src/routes/jira/jira-get-connected-repos.ts

```diff
@@ -0,0 +1,139 @@
+import { NextFunction, Request, Response } from "express";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+import { Subscription, TaskStatus } from "~/src/models/subscription";
+
+interface Page {
+	pageNum: number;
+	isCurrentPage: boolean;
+}
+
+export const JiraGetConnectedRepos = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+
+	try {
+		const { jiraHost, nonce } = res.locals;
+		const subscriptionId = Number(req.params.subscriptionId) || Number(req.body.subscriptionId);
```

Comment:
Why do we need both params?

Reply:
I genuinely cant remember why we need both, i guess one is a safety net potentially coming from different routes.